src/contactCard/Summary.tsx

```diff
@@ -18,10 +18,19 @@ export function renderSummary(
             {renderContactSummary(profile, onContactDetailsClick)}
             {renderOrgSummary(manager, isManagerLoading, onOrgDetailsClick, onPersonaClick)}
         </ul>
-    );
-}
-
+    ); 
+    }
 
+    let isExpanded = false;
+    function handleContactDetailsClick(onContactDetailsClick: () => void) {
+        onContactDetailsClick();
+        const button = document.querySelector('[data-focus="button"]');
+        if (button) {
+          (button as HTMLButtonElement).focus();
+        }
+        isExpanded = true;
```

Comment:
You should never mutate variables directly in React. This should be changed into a state variable lifted up to the parent component and then passed down as a prop to renderContactSummary.